### ISSUE TITLE
Sourcemap warnings

### ISSUE THREAD

**Issue by chrism** [sir-dunxalot/ember-tooltips#226]
I'm using the 3.0.0.beta-2 version and get these warnings whenever I start the server:

```sh
Warning: ignoring input sourcemap for vendor/popper/popper.js because ENOENT: no such file or directory, open '/Users/username/git/project/tmp/source_map_concat-input_base_path-9wXo8Gz4.tmp/vendor/popper/popper.js.map'
Warning: ignoring input sourcemap for vendor/popper/tooltip.js because ENOENT: no such file or directory, open '/Users/username/git/project/tmp/source_map_concat-input_base_path-9wXo8Gz4.tmp/vendor/popper/tooltip.js.map'
```

Is there a way to suppress/remove them?

Thanks—and thanks for the great library.

**Comment by allthesignals** [comment]
I'm seeing this too - we are also seeing `Cannot read property 'popper' of undefined`. For context, we are including this as a dependency in an addon. cc @andycochran

**Comment by maxfierke** [comment]
This is fixed in 3.0.0-beta.3. Thanks for the report!
